### DISCUSSION TITLE
Get a logo

### DISCUSSION THREAD

**Discussion by maelle** [cynkra/dm#1373] [General]

**Comment by krlmlr** [comment]
https://github.com/cynkra/dm/pull/1067#issuecomment-1154160500

**Comment by maelle** [comment]
How do you get this dark mode, is it a browser extension?

**Comment by krlmlr** [comment]
DarkReader in Firefox.

**Comment by krlmlr** [comment]
I don't mind using the flights data as a logo for now, perhaps reassess the color of the `airports` table?

**Comment by maelle** [comment]
I'll look into it next week then.

For some reason this current logo idea reminds me of airplanes with flags (with messages / ads).

**Comment by maelle** [comment]
A real hex logo would be nice. :thinking:

**Comment by maelle** [comment]
Using the diagram without text (keeping this code snippet here for reference, it's part of the "cheatsheet project")

```r
library("dm")
xml <- dm_nycflights13() |>
  dm_draw() |>
  DiagrammeRsvg::export_svg() |>
  xml2::read_xml()

text_nodes <- xml2::xml_find_all(xml, ".//d1:text")
xml2::xml_text(text_nodes) <- ""

xml2::write_xml(xml, "logo.svg")
```
![logo](https://user-images.githubusercontent.com/8360597/178488959-8014d24f-ba58-447c-b96a-7c0f5181d1bc.svg)

**Comment by moodymudskipper** [comment]
FWIW I took a very similar approach for {flow}'s hex logo : 

![image](https://user-images.githubusercontent.com/18351714/178580098-ca34c77c-5bd2-4a38-85fd-ee8e1f167d75.png)

The database icon might be used too, we could imagine something like below but with the database symbol in the background, (lighter basically), wearing the pkg name, not linked to tables, and table centred on top of it and linked between each other, with little key icons if we want to be fancy.

![image](https://user-images.githubusercontent.com/18351714/178580955-96f50210-e7a5-464f-86fb-1b9f4e241a0e.png)

**Comment by IndrajeetPatil** [comment]
WDYT? Am I on the right track here?

![image](https://user-images.githubusercontent.com/11330453/184705434-73235315-61bf-4efb-8d3e-c89335115ee6.png)

**Comment by krlmlr** [comment]
Thanks for your contributions. Hex stickers/logos are currently out of scope, we might revisit later.
